app/services/github_service.rb

```diff
@@ -9,4 +9,14 @@ def self.followers(current_user)
     response = Faraday.get("https://api.github.com/users/#{current_user.username}/followers?client_id=#{ENV['github_client_id']}&client_secret=#{ENV['github_client_secret']}")
     JSON.parse(response.body, symbolize_names: true)
   end
+
+  def self.following(current_user)
+    response = Faraday.get("https://api.github.com/users/#{current_user.username}/following?cliend_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
@s-espinosa I saw some examples where people were setting instance variables for the first part of the url and then tacking on the endpoint in the class method. I'm going to add an example.

initialize
@connection = Faraday.new('https://api.github.com')

def self.repos(current_user)
  parse(connection.get("/users/#{current_user.username}/repos?client_id=#{ENV['github_client_id']}&client_secret=#{ENV['github_client_secret']}")
end